canvas_modules/common-canvas/__tests__/common-properties/components/editor-form-test.js

```diff
@@ -126,53 +147,56 @@ describe("Right flyout category views", () => {
+		const renderedObject = propertyUtilsRTL.flyoutEditorForm(DATEFIELD_PARAM_DEF, { containerType: "Custom", rightFlyout: true, categoryView: "accordions" });
 		wrapper = renderedObject.wrapper;
-		const editorForm = wrapper.find(".properties-editor-form");
-		expect(editorForm.prop("className").includes("right-flyout-tabs-view")).to.equal(false);
+		const { container } = wrapper;
+		const editorForm = container.getElementsByClassName("properties-editor-form")[0];
+		expect(editorForm.className.includes("right-flyout-tabs-view")).to.equal(false);
 
-		const propertiesCategories = editorForm.find(".properties-categories");
+		const propertiesCategories = editorForm.getElementsByClassName("properties-categories");
 		expect(propertiesCategories).to.have.length(1);
 
-		const categoryContainers = propertiesCategories.find(".properties-category-container");
+		const categoryContainers = editorForm.getElementsByClassName("properties-category-container");
 		expect(categoryContainers).to.have.length(3);
 
 		// Verify properties title has bottom border
-		const titleEditor = wrapper.find(".properties-title-editor");
-		expect(titleEditor.prop("className").includes("properties-title-right-flyout-tabs-view")).to.equal(false);
+		const titleEditor = container.getElementsByClassName("properties-title-editor")[0];
+		expect(titleEditor.className.includes("properties-title-right-flyout-tabs-view")).to.equal(false);
 	});
 
 	it("For custom container in right flyout, when categoryView is not set, categories should be displayed as accordions by default", () => {
-		const renderedObject = propertyUtils.flyoutEditorForm(DATEFIELD_PARAM_DEF, { containerType: "Custom", rightFlyout: true });
+		const renderedObject = propertyUtilsRTL.flyoutEditorForm(DATEFIELD_PARAM_DEF, { containerType: "Custom", rightFlyout: true });
 		wrapper = renderedObject.wrapper;
-		const editorForm = wrapper.find(".properties-editor-form");
-		expect(editorForm.prop("className").includes("right-flyout-tabs-view")).to.equal(false);
+		const { container } = wrapper;
+		const editorForm = container.getElementsByClassName("properties-editor-form")[0];
+		expect(editorForm.className.includes("right-flyout-tabs-view")).to.equal(false);
 
-		const propertiesCategories = editorForm.find(".properties-categories");
+		const propertiesCategories = editorForm.getElementsByClassName("properties-categories");
 		expect(propertiesCategories).to.have.length(1);
 
-		const categoryContainers = propertiesCategories.find(".properties-category-container");
+		const categoryContainers = editorForm.getElementsByClassName("properties-category-container");
 		expect(categoryContainers).to.have.length(3);
 
 		// Verify properties title has bottom border
-		const titleEditor = wrapper.find(".properties-title-editor");
-		expect(titleEditor.prop("className").includes("properties-title-right-flyout-tabs-view")).to.equal(false);
+		const titleEditor = container.getElementsByClassName("properties-title-editor")[0];
```

Comment:
If there is only one element then you can also use `container.querySelector(".classname")` to make it more compact, but above is also fine.

Reply:
Yea I'll switch it to this